integration_tests/snaps/simple-rust-with-revision/snapcraft.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-rust-on-revision
```

Comment:
I found this week that I dislike that we are calling everything simple-*
Instead, it would be easier to find things if we called them plugin-*
Can you rename this dir to something like integration_tests/snaps/rust-with-revision ?